test/integration/npm/main.go

```diff
@@ -162,6 +163,9 @@ func main() {
 	printAndWait(true)
 	panicOnError(dp.AddPolicy(policies.TestNetworkPolicies[0]))
 	fmt.Println("AZURE-NPM should have rules now")
+
+	fmt.Println("waiting for reconcile to finish (will be a while if you don't update the reconcile time in policymanager.go")
+	time.Sleep(finalWaitTimeInMinutes * time.Minute)
```

Comment:
Why do we need this? Is there a channel we could introduce into the code that would signal that things are done instead? This slows down the integration tests

Reply:
we don't run these in our pipeline (I just run it locally), so speed shouldn't be a problem